data/default_release_images.json

```diff
@@ -29,5 +29,11 @@
         "cpu_architecture": "arm64",
         "url": "quay.io/openshift-release-dev/ocp-release:4.9.13-aarch64",
         "version": "4.9.13"
+    },
+    {
+        "openshift_version": "4.10",
+        "cpu_architecture": "x86_64",
+        "url": "quay.io/openshift-release-dev/ocp-release:4.10.0-fc.0-x86_64",
+        "version": "4.10.0"
```

Comment:
it should be something like "4.10.0-fc.0". we use the postfix as an indication for support level: https://github.com/openshift/assisted-service/blob/master/internal/versions/versions.go#L337